pkg/services/govmomi/util.go

```diff
@@ -525,7 +533,14 @@ func waitForIPAddresses(
 			// If the device spec requires static IP addresses, the wait
 			// is not over if the device lacks one of those addresses.
 			for _, specIP := range deviceSpec.IPAddrs {
-				if _, ok := macToHasStaticIP[mac][specIP]; !ok {
+				ip, _, err := gonet.ParseCIDR(specIP)
+				if err != nil {
```

Comment:
If we do #853, getting an error here should be impossible

Reply:
#853 is going to be merged today, so I'd rather just remove this

Reply:
Ack, will remove the error check.